dropbox/oauth.py

```diff
@@ -120,11 +130,18 @@ def __repr__(self):
 
 class DropboxOAuth2FlowBase(object):
 
-    def __init__(self, consumer_key, consumer_secret, locale=None, token_access_type='legacy',
-                 scope=None, include_granted_scopes=None):
+    def __init__(self, consumer_key, consumer_secret=None, locale=None, token_access_type='legacy',
+                 scope=None, include_granted_scopes=None, pkce_method=None):
         if scope is not None:
             assert len(scope) > 0 and isinstance(scope, list), \
```

Comment:
BTW I think it is time to point out that.

Assertion Error should be use when you are absolutely sure that the input can't be False. For this type of user error, we should throw other exceptions.